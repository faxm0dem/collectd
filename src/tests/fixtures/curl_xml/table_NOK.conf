LoadPlugin "curl_xml"

<Plugin "curl_xml">
  <URL "file://table.xml">
    Host "gemaster"
    Instance "GridEngine"
    <xpath "/GridEngineBasicCount/Jobs">
      Type "gauge"
      ValuesFrom "Count/text()"
    </xpath>
  </URL>
</Plugin>

