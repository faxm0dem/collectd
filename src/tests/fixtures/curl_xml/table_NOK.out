[error] curl_xml plugin: InstanceFrom is must in xpath block since the base xpath expression "/GridEngineBasicCount/Jobs" returned multiple results. Skipping the xpath block...
